sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -650,6 +650,7 @@ object FunctionRegistry {
     expression[Now]("now"),
     expression[Quarter]("quarter"),
     expressionBuilder("second", SecondExpressionBuilder),
+    expression[SecondsOfTimeWithFraction]("secondoftime_with_fraction"),
```

Comment:
Please, don't expose the expression with such name to the public API so far.

Reply:
Removed the function registration